e2e/terraform/packer/packer-windows.json

```diff
@@ -44,14 +44,24 @@
     {
       "type": "windows-restart"
     },
+    {
+      "type": "file",
+      "source": "./windows/install-nomad.ps1",
+      "destination": "/opt/install-nomad.ps1"
+    },
+    {
+      "type": "powershell",
+      "elevated_user": "Administrator",
+      "elevated_password": "{{.WinRMPassword}}",
+      "inline": ["/opt/install-nomad.ps1 -nomad_version 0.9.6 -nostart"]
```

Comment:
A bit surprised to see 0.9.6 here?  Should we pick a more modern version or is that intended to be parameterized?

Reply:
I'm inclined to agree with you, but that's what we're currently doing in the AMI builds. I'm intentionally not changing that in this PR so that it's touching one thing at a time.